libavfilter/vf_drawbox.c

```diff
@@ -81,6 +82,7 @@ typedef struct DrawBoxContext {
     char *t_expr;          ///< expression for thickness
     int have_alpha;
     int replace;
+    int side_data;
```

Comment:
use another name such as 'box_from' or 'box_source' or 'box_sidedata', and the value may be of type AVFrameSideDataType, so we can draw box for boudingbox or AV_FRAME_DATA_REGIONS_OF_INTEREST.

btw, i have a v5 patch just now, it does not use private_ref, just wait for community's comment.